python/lsst/afw/table/exposure/exposure.cc

```diff
@@ -176,29 +141,22 @@ void declareExposureCatalog(PyExposureCatalog & cls) {
             (Catalog (Catalog::*)(geom::Point2D const &, image::Wcs const &, bool) const)
              &Catalog::subsetContaining,
             "point"_a, "wcs"_a, "includeValidPolygon"_a=false);
+    return cls;
 };
 
 }  // anonymous namespace
 
 
-PYBIND11_PLUGIN(_exposure) {
-    py::module mod("_exposure", "Python wrapper for afw _exposure library");
-
-    /* Module level */
-    PyExposureRecord clsExposureRecord(mod, "ExposureRecord");
-    PyExposureTable clsExposureTable(mod, "ExposureTable");
-    PyExposureColumnView clsExposureColumnView(mod, "ExposureColumnView");
-    PyBaseExposureCatalog clsBaseExposureCatalog(mod, "_BaseExposureCatalog");
-    PySortedBaseExposureCatalog clsSortedBaseExposureCatalog(mod, "_SortedBaseExposureCatalog");
-    PyExposureCatalog clsExposureCatalog(mod, "ExposureCatalog", py::dynamic_attr());
-
-    /* Members */
-    declareExposureRecord(clsExposureRecord);
-    declareExposureTable(clsExposureTable);
-    table::python::declareColumnView(clsExposureColumnView);
-    table::python::declareCatalog(clsBaseExposureCatalog);
-    table::python::declareSortedCatalog(clsSortedBaseExposureCatalog);
-    declareExposureCatalog(clsExposureCatalog);
+PYBIND11_PLUGIN(exposure) {
+    py::module mod("exposure");
+    py::module::import("lsst.afw.table.simple");
+    py::module::import("lsst.afw.geom");
+    // afw.image and afw.detection cannot be imported due to circular dependencies
```

Comment:
They are not imported in Python, and I believe they cannot be unless it's in `afw/__init__.py`, which I'm not sure we want to modify.  This is a defect of our poor in-afw package organization.